modules/platform.py

```diff
@@ -1201,3 +1201,30 @@ def get_feed_by_name(self, name=None):
                     return {}
         else:
             raise HTTPException(response)
+
+
+class V3_latest(VectraPlatformClientV3_3):
```

Comment:
The naming convention is odd for this class.  The class name should ideally give you a good idea of what is being instantiated.  I recommend ClientV3_latest, which would match the existing pattern you have.

Reply:
Updated the class name to ClientV3_latest